pkg/anypoint/client.go

```diff
@@ -296,11 +296,11 @@ func (c *AnypointClient) GetAPI(id string) (*API, error) {
 
 // GetPolicies lists the API policies.
 func (c *AnypointClient) GetPolicies(apiID int64) ([]Policy, error) {
-	policies := []Policy{}
+	policies := Policies{}
 	url := fmt.Sprintf("%s/apimanager/api/v1/organizations/%s/environments/%s/apis/%d/policies", c.baseURL, c.auth.GetOrgID(), c.environment.ID, apiID)
 	err := c.invokeJSONGet(url, nil, &policies, nil)
```

Comment:
I think we need to do a check for err here, if there is an err then try to get the policies with the []Policy type and return that array. Seems there were versions of mulesoft that return the array of policies and another that returns an object containing an array of policies

Reply:
This still needs to be handled, no? I know we discussed not worry about other issues, but this may be one that we should handle

Reply:
never mind I see it now...duh